### PR TITLE
handle call when multiple devices are involved for one account

### PR DESCRIPTION
**Pull request by coder-with-a-bushido**
Closes https://github.com/famedly/product-management/issues/1512

### CODE REVIEW

File: lib/src/voip/voip.dart [hunk 1]
```diff
@@ -366,6 +373,11 @@ class VoIP {
             'Ignoring call reject for room $roomId claiming to be for call in room ${call.room.id}');
         return;
       }
+      if (call.remotePartyId != partyId) {
```
Comment (techno-disaster) [inline lib/src/voip/voip.dart:376]: remotePartyId is only set when there you already get a answer, if there was no answer, this would break ending calls

**Review by techno-disaster — CHANGES_REQUESTED** [review]

File: lib/src/voip/voip.dart (hunk 1 above)
Comment (coder-with-a-bushido) [reply lib/src/voip/voip.dart]: added a null check before that

**Review by techno-disaster — APPROVED** [review]
LGTM

**Review by techno-disaster — CHANGES_REQUESTED** [review]
Can you quickly change the issue description so that we don't completely close the linked issue?

**Review by td-famedly — APPROVED** [review]

**Pull request merged**